### ISSUE TITLE
Different output

### ISSUE THREAD

**Issue by HDaoud** [openalpr/openalpr#358]
Hello Mathew,

I prepared a new country `my` for Malaysian Plates, however, I noticed something related to code output depending on the way you query `alpr`:

first command without `*` after the `Examples/` it miss one plate result for file **02-03-16_13_04_01.jpg**
When querying the file itself only or `Examples/*` the results showup!

The other question is why this faulty 1111 is shown, in `eu` this is never shown I believe.

>:~$ alpr -c my ~/Documents/OpenALPR/Car\ Images/Examples/
/home/hdaoud/Documents/OpenALPR/Car Images/Examples//02-03-16_12_58_42.jpg
plate0: 10 results
    - WC1987N	 confidence: 87.2941
    - C1987N	 confidence: 78.5535
    - WC187N	 confidence: 78.1551
    - WC1S87N	 confidence: 77.9428
    - WC1B87N	 confidence: 76.234
    - W01987N	 confidence: 73.4544
    - W1987N	 confidence: 72.4523
    - C187N	 confidence: 69.4145
    - C1S87N	 confidence: 69.2022
    - C1B87N	 confidence: 67.4934
/home/hdaoud/Documents/OpenALPR/Car Images/Examples//02-03-16_13_01_33.jpg
plate0: 10 results
    - WXX5031	 confidence: 90.1591
    - WXX5Q31	 confidence: 80.9922
    - WXX50S1	 confidence: 79.0307
    - WXX501	 confidence: 78.3779
    - WXX5D31	 confidence: 78.3727
    - WXX531	 confidence: 77.1574
    - WXX5QS1	 confidence: 69.8638
    - WXX5Q1	 confidence: 69.211
    - WXX5DS1	 confidence: 67.2443
    - WXX5D1	 confidence: 66.5915
/home/hdaoud/Documents/OpenALPR/Car Images/Examples//02-03-16_13_04_01.jpg
plate0: 10 results
    - 1111	 confidence: 77.1315
    - 11111	 confidence: 72.5611
    - 1U111	 confidence: 67.4481
    - 1H111	 confidence: 66.1648
    - U111	 confidence: 65.2836
    - 111111	 confidence: 64.241
    - H111	 confidence: 64.0004
    - 11U111	 confidence: 62.8777
    - 1U11	 confidence: 62.0405
    - 11H111	 confidence: 61.5944
/home/hdaoud/Documents/OpenALPR/Car Images/Examples//02-03-16_13_04_41.jpg
plate0: 10 results
    - CCW967	 confidence: 81.8742
    - CCW9677	 confidence: 79.1576
    - CCW96	 confidence: 77.4302
    - CCWS67	 confidence: 73.2326
    - CCW67	 confidence: 72.9801
    - CC967	 confidence: 71.0945
    - CCWS677	 confidence: 70.5159
    - CCW677	 confidence: 70.2635
    - CCWB67	 confidence: 70.0643
    - C0W967	 confidence: 69.0802
hdaoud@hdaoud-Dell-Precision-M3800:~$ alpr -c my ~/Documents/OpenALPR/Car\ Images/Examples/*
plate0: 10 results
    - WC1987N	 confidence: 87.2941
    - C1987N	 confidence: 78.5535
    - WC187N	 confidence: 78.1551
    - WC1S87N	 confidence: 77.9428
    - WC1B87N	 confidence: 76.234
    - W01987N	 confidence: 73.4544
    - W1987N	 confidence: 72.4523
    - C187N	 confidence: 69.4145
    - C1S87N	 confidence: 69.2022
    - C1B87N	 confidence: 67.4934
plate0: 10 results
    - WXX5031	 confidence: 90.1591
    - WXX5Q31	 confidence: 80.9922
    - WXX50S1	 confidence: 79.0307
    - WXX501	 confidence: 78.3779
    - WXX5D31	 confidence: 78.3727
    - WXX531	 confidence: 77.1574
    - WXX5QS1	 confidence: 69.8638
    - WXX5Q1	 confidence: 69.211
    - WXX5DS1	 confidence: 67.2443
    - WXX5D1	 confidence: 66.5915
**plate0: 10 results**
    **- CCQ7637	 confidence: 86.0843**
    - CCQ7637E	 confidence: 83.365
    - CCQ7637G	 confidence: 82.7665
    - CC07637	 confidence: 79.4127
    - CC7637	 confidence: 76.9811
    - CCD7637	 confidence: 76.8382
    - CC07637E	 confidence: 76.6934
    - CCQ637	 confidence: 76.5596
    - CC07637G	 confidence: 76.0949
    - CCQ737	 confidence: 75.9626
**plate1: 10 results**
    - 1111	 confidence: 77.1315
    - 11111	 confidence: 72.5611
    - 1U111	 confidence: 67.4481
    - 1H111	 confidence: 66.1648
    - U111	 confidence: 65.2836
    - 111111	 confidence: 64.241
    - H111	 confidence: 64.0004
    - 11U111	 confidence: 62.8777
    - 1U11	 confidence: 62.0405
    - 11H111	 confidence: 61.5944
plate0: 10 results
    - CCW967	 confidence: 81.8742
    - CCW9677	 confidence: 79.1576
    - CCW96	 confidence: 77.4302
    - CCWS67	 confidence: 73.2326
    - CCW67	 confidence: 72.9801
    - CC967	 confidence: 71.0945
    - CCWS677	 confidence: 70.5159
    - CCW677	 confidence: 70.2635
    - CCWB67	 confidence: 70.0643
    - C0W967	 confidence: 69.0802

Thanks!

**Comment by matthill** [comment]
Yes, I've noticed there's some state stored in the OpenCV LBP detector.  When you initialize from scratch, results are always deterministic, but when you run one after another (e.g., reading all images in a directory, or a video stream) there's some randomness.

**Comment by alexkiddo** [comment]
Hi Hdaoud could you share the results of your training for MY plates?

**Comment by HDaoud** [comment]
Hi @alexkiddo It's not that perfect yet, the training dataset that I've got is considered small, if you have one kindly let me know, so I can retrain to obtain a better results, sufficient to be shared!

**Comment by hanzhawary** [comment]
hi @HDaoud its being a long day since i started using openALPR and bumped with this tread, would you mind to tell how to get that malaysia dataset of license plate recognition? i've been trying using given default country and nearest country to mine which is Singapore, but no luck with malaysia. May i know the way to solve this issue?

regards,
Muhammad Hanif Al-Hawari Al-Malek Faseh

**Comment by jerrychong25** [comment]
Hi @hanzhawary,

Same here.

I would like to know too :)
